modules/core/src/copy.cpp

```diff
@@ -599,6 +867,26 @@ flipVert( const uchar* src0, size_t sstep, uchar* dst0, size_t dstep, Size size,
         int i = 0;
         if( ((size_t)src0|(size_t)dst0|(size_t)src1|(size_t)dst1) % sizeof(int) == 0 )
         {
+#if CV_SIMD
+#if CV_SIMD256 || CV_SIMD512
```

Comment:
Does tail processing make essential difference? It's possible to process half nlanes elements by `vx_load_low` `vx_store_low` in this case to reduce the tail size. Also check should be `CV_SIMD_WIDTH > 16` in this case to ensure everything will be fine if someday `CV_SIMD1024` appears

Reply:
Simplifying into native vx_load/vx_store case.